### PR TITLE
Tutorial: Install TYPO3 on DDEV

### PR DESCRIPTION
**Pull request by linawolf**
releases: main, 11.5

### CODE REVIEW

File: Documentation/Installation/Index.rst
Comment (brotkrueml) [inline Documentation/Installation/Index.rst:37]:
```suggestion
            DDEV, Docker and Composer.
```

As it writes itself: https://ddev.readthedocs.io/en/stable/

File: Documentation/Installation/TutorialDdev.rst
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:67]:
```suggestion
            Docroot Location (current directory): public
```

File: Documentation/Installation/TutorialDdev.rst
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:121]: Maybe link "system maintainer", not everyone may know, what that is: https://docs.typo3.org/m/typo3/tutorial-getting-started/11.5/en-us/UserManagement/BackendPrivileges/Index.html#system-maintainers

File: Documentation/Installation/TutorialDdev.rst
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:178]: This sentence sounds strange. Maybe: "If you want to stop all DDEV projects ..."

File: Documentation/Installation/TutorialDdev.rst
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:186]:
```suggestion
If you want to delete the project you just created you can delete it by
```
A little bit "delete" too often.

File: Documentation/Installation/TutorialDdev.rst
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:187]:
```suggestion
calling the following command in your new projects root folder:
```

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/Installation/TutorialDdev.rst
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:178]:
```suggestion
If you want to stop all projects from running on DDEV you can call:
```

File: Documentation/Installation/TutorialDdev.rst
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:121]:
```suggestion
        (who is also :ref:`system maintainer <system-maintainer>`).
```

File: Documentation/Installation/TutorialDdev.rst
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:19]:
```suggestion
Preparations to install DDEV
```

File: Documentation/Installation/TutorialDdev.rst
Comment (brotkrueml) [inline Documentation/Installation/TutorialDdev.rst:130]:
```suggestion
        #.  Create a file called :file:`FIRST_INSTALL` in your webroot
```

**Review by brotkrueml — APPROVED** [review]

**Comment by linawolf** [comment]
Let us wait for the review of @tomwarwick please

File: Documentation/Installation/TutorialDdev.rst
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:151]: Actually DDEV creates this file. If there would have been an AdditionalConfiguration already, DDEV would not have modified it

File: Documentation/Installation/TutorialDdev.rst
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:167]: shall we use plural databases? Each project has its own database

File: Documentation/Installation/TutorialDdev.rst
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:172]: delete.. delete... the sentence sounds strange...

File: Documentation/Installation/TutorialDdev.rst
Comment (linawolf) [inline Documentation/Installation/TutorialDdev.rst:142]: Line to long

**Review by tomwarwick — APPROVED** [review]

**Pull request merged**
